services/web/pkg/apps/apps.go

```diff
@@ -107,7 +101,7 @@ func List(logger log.Logger, data map[string]config.App, fSystems ...fs.FS) []Ap
 	return maps.Values(registry)
 }
 
-func Build(fSystem fs.FS, id string, conf map[string]any) (Application, error) {
+func build(fSystem fs.FS, id string, conf map[string]any) (Application, error) {
```

Comment:
not against it to make it private, but may i ask why?

Reply:
It is not used outside of the package.